types.ts

```diff
@@ -413,6 +418,30 @@ export interface WebBasicAuthentication extends BaseAuthentication {
   };
 }
 
+export interface CustomAuthParameter {
+  /**
+   * The name used to refer to the parameter. Should match the string for the key in the params mapping.
```

Comment:
Not sure what the "params mapping" is. Maybe `A name used to refer to this parameter. Use this name to generate the template replacement string. See {@link CustomAuthentication}.`

Reply:
added link to the overall CustomAuthParameter doc string and updated to reflect your sentiment more